jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererAutoTriggerService.kt

```diff
@@ -171,36 +172,61 @@ class CodeWhispererAutoTriggerService : CodeWhispererAutoTriggerHandler, Disposa
 
         val rightContext = caretContext.rightFileContext
         val lengthOfRight = rightContext.trim().length
-        val triggerTypeCoefficient = CodeWhispererClassifierConstants.triggerTypeCoefficientMap[automatedTriggerType] ?: 0.0
+
+        val isExperimentGroup = CodeWhispererUserGroupSettings.getInstance().getUserGroup() == CodeWhispererUserGroup.Classifier
```

Comment:
In what condition will Classifier be assigned to a specific user?

Reply:
1/3 random chance for a user to be in this group